docs/faq.md

```diff
@@ -0,0 +1,53 @@
+# Frequently Asked Questions
+
+- [Main.py Not Working](#Main.py-is-not-working)
+- [Easy Way To Edit Config](#easy-way-to-edit-config)
+- [Read PThe Post Not Just The Title](#some-of-my-markdown-elements-arent-highlighted)
+- [TikTok Reader Crashing](#tiktok-reader-crashing)
+- [Preview TTS Voice](#preview-tts-voice)
+- [Could Not Concat Numpy.float64](#could-not-concat-numpyfloat64)
+- [Video Length Is Too Long & I Am Too Lazy To Edit It Myself](#video-length-is-too-long--i-am-too-lazy-to-edit-it-myself)
+- [TikTok SessionId Expiring](#tiktok-sessionid-expiring)
+- [Main.py Crashing Due To Video Length](#mainpy-crashing-due-to-video-length)
+
+## Mainpy is not working
+
+The most common occurence is the use of an incorrect Python version, or a missing dependency. Make sure you have Python 3.10 installed, and that you have installed the dependencies using `pip install -r requirements.txt`. If you are still having trouble, feel free to ask for help in the [Discord](https://discord.gg/QaTx2ZDqea)
+
+## Easy Way To Edit Config
+
+Run `GUI.py` this opens a web editor in which you can edit the config file. You can also preview the TTS voices.
+
+## Read The Post Not Just The Title
+
+The bot by default will only read the title of the post. If you want to read the post as well, you can change the `storymode` option in the `config.toml` file to `true`. This will make the bot read the post as well as the title.
```

Comment:
```suggestion
The bot by default will only read the title of the post. If you want to read the post as well, you can change the `story_mode` option in the `config.toml` file to `true`. This will make the bot read the post as well as the title. Method 0 is one screenshot of the whole text. We recommend method 1, which is a fancy image for every sentence.
```